RELEASE_PROCESS.md

```diff
@@ -0,0 +1,118 @@
+
+For details on ICS releases, see [RELEASES.md](./RELEASES.md).
+
+## Changelog
+
+For PRs that are changing production code, please add a changelog entry in `.changelog` (for details, see [contributing guidelines](./CONTRIBUTING.md#changelog)). 
+
+To manage and generate the changelog on ICS, we currently use [unclog](https://github.com/informalsystems/unclog). 
+Read the [README.md](https://github.com/informalsystems/unclog#readme) in the unclog repo for instruction on how to install and use unclog.
+
+### Creating a new release branch 
+
+Unreleased changes are collected on `main` in `.changelog/unreleased/`. 
+However, `.changelog/` on `main` contains also existing releases (e.g., `v3.2.0`).
+Thus, when creating a new release branch (e.g., `release/v3.3.x`), the following steps are necessary:
+
+- create a new release branch, e.g., `release/v3.3.x`
+    ```bash 
+    git checkout main
+    git pull 
+    git checkout -b release/v3.3.x
+    ```
+- delete all the sub-folders in `.changelog/` except `unreleased/` 
+    ```bash
+    find ./.changelog -mindepth 1 -maxdepth 1 -type d -not -name unreleased | xargs rm -r
+    ```
+- replace the content of `.changelog/epilogue.md` with the following text
+    ```md
+    ## Previous Versions
+
+    [CHANGELOG of previous versions](https://github.com/cosmos/interchain-security/blob/main/CHANGELOG.md)
+    ```
+- push the release branch upstream 
+    ```bash 
+    git push
+    ```
+
+### Cutting a new release
+
+Before cutting a _**release candidate**_ (e.g., `v3.3.0-rc0`), the following steps are necessary:
+
+- move to the release branch, e.g., `release/v3.3.x`
+    ```bash 
+    git checkout release/v3.3.x
+    ```
+- move all entries in ".changelog/unreleased" to the release version, e.g., `v3.3.0`, i.e.,
+    ```bash
+    unclog release v3.3.0
+    ```
+- update `CHANGELOG.md`, i.e.,
+    ```bash
+    unclog build > CHANGELOG.md
+    ```
+- open a PR (from this new created branch) against the release branch, e.g., `release/v3.3.x`
+
+Now you can cut the release candidate, e.g., v3.3.0-rc0 (follow the [Tagging Procedure](#tagging-procedure)).
+
+### Update the changelog on main
+
+Once the **final release** is cut, the new changelog section must be added to main:
+
+- checkout a new branch from the `main` branch, i.e.,
+    ```bash
+    git checkout main
+    git pull 
+    git checkout -b <username>/backport_changelog
+    ```
+- bring the new changelog section from the release branch into this branch, e.g.,
+    ```bash
+    git checkout release/v3.3.x .changelog/v3.3.0
+    ```
+- remove duplicate entries that are both in `.changelog/unreleased/` and the new changelog section, e.g., `.changelog/v3.3.0`
+- update `CHANGELOG.md`, i.e.,
+    ```bash
+    unclog build > CHANGELOG.md
+    ```
+- open a PR (from this new created branch) against `main`
+
+## Tagging Procedure
+
+**Important**: _**Always create tags from your local machine**_ since all release tags should be signed and annotated.
```

Comment:
Could you make `signed and annotated` into a link to GH docs on commit signing?
* https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits